README.md

```diff
@@ -54,22 +54,40 @@ C:\vcpkg> vcpkg install wil:x64-windows
 Note that even though WIL is a header-only library, you still need to install the package for all architectures/platforms you wish to use it with. Otherwise, WIL won't be added to the include path for the missing architectures/platforms. Execute `vcpkg help triplet` for a list of available options.
 
 # Building/Testing
-To get started testing WIL, first make sure that you have a recent version of [Visual Studio](https://visualstudio.microsoft.com/downloads/) and the most recent [Windows SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk) installed. If you are doing
-any non-trivial work, also be sure to have a recent version of [Clang](http://releases.llvm.org/download.html) installed. Once everything is installed, open a VS
-native command window (e.g. "x64 Native Tools Command Prompt for VS 2022"). If you are familiar with CMake you can get started building normally. Otherwise, or if you prefer to skip all of the boilerplate, you can use one of the scripts in the [scripts](scripts) directory:
-```cmd
-C:\wil> scripts\init.cmd -c clang -g ninja -b debug
-```
-You can execute `init.cmd --help` for a summary of available options. The scripts use a common directory pattern of `build/$(compiler)$(arch)$(type)` for the build output root. E.g. `build/clang64debug` when using Clang as the compiler, x64 as the architecture, and Debug as the build type. It is this directory where you will want to build from. For example, if you initialized using the command above, you can build the tests like so:
+
```

Comment:
I've had great luck doing the "open folder with existing code" option in very recent VS builds - the cmake integration picks up the configurations and defaults to x64-debug so it's _nearly_ "load and hit F5."

Reply:
I gave this a shot and found:

* IntelliSense works flawlessly.
* Building "work" but does not detect the test executables. Not sure what it ends up building.
* F5 crashes on launch. Not sure what it's launching
* Cannot run tests. They are not discovered in Test Explorer.

versus the generated `.sln` which correctly builds and can debug. I documented as such.

Was that your experience? I'm happy to update if you have a better method, or I'm happy to let you update.